website/src/pages/index.tsx

```diff
@@ -80,7 +80,7 @@ const QUOTES = [
 
 function Home() {
   const context = useDocusaurusContext();
-  const {siteConfig: {customFields = {}, tagline} = {}} = context;
+  const {siteConfig: {customFields = {}, tagline = ''} = {}} = context;
```

Comment:
It won't hurt to remove them on our site, definitely. I guess we should instead work on improving the `useDocusaurusContext()` typing because right now it just returns `any`